generate-deploy-script.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+if [ -n "$DEBUG" ]; then
+	set -x
+fi
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+DIR=$(cd $(dirname "${BASH_SOURCE}") && pwd -P)
+
+RELEASE_NAME=timescale-observability
+NAMESPACE=timescale-observability
```

Comment:
With timescale-observability as the release name, some of the resources may end up having names > 63 characters and k8s will not deploy them
With all the resources already in the timescale-obs namespace we may opt for a smaller release name

Reply:
Ah didn't know that... So we should be using `timescale-obs`? I guess `ts-obs` is to short :grinning:

Reply:
This is the example case for this error. Just tried with `0.1.0-alpha.3`  release
```
$ helm install timescale-observability timescale/timescale-observability --devel

Error: CronJob.batch "timescale-observability-timescale-prometheus-drop-chunk" is 
invalid: metadata.name: Invalid value: "timescale-observability-timescale-prometheus-drop-chunk": 
must be no more than 52 characters
```

Reply:
let's do `ts-observability`?

Reply:
but we should also fix this issue

Reply:
https://github.com/helm/helm/issues/6006 it seems helm is the source of the error, not k8s, a old check for older k8s versions. But they haven't removed it